installing/installing_restricted_networks/installing-disconnected.adoc

```diff
@@ -32,6 +32,9 @@ include::modules/installation-configuration-parameters.adoc[leveloffset=+2]
 
 include::modules/installation-azure-config-yaml.adoc[leveloffset=+2]
 
+// TODO: If this assembly is going to be for restricted network Azure install, whenever it is included, need to confirm whether proxy is supported before including the following file:
```

Comment:
Restricted network Azure isn't in scope for 4.2, and I think I need to remove this file.

Reply:
@kalexand-rh Yeah, I heard it wasn't yet, but I wasn't sure if you were just prepping with this file for whenever it will be, that's why I added the note.